gluten-core/src/main/scala/io/glutenproject/execution/HashJoinExecTransformer.scala

```diff
@@ -248,26 +136,16 @@ trait HashJoinLikeExecTransformer
     "hashBuildSpilledFiles" -> SQLMetrics.createMetric(
       sparkContext, "total spilled files of hash build"),
 
-    "hashProbeInputRows" -> SQLMetrics.createMetric(
-      sparkContext, "number of hash probe input rows"),
-    "hashProbeInputVectors" -> SQLMetrics.createMetric(
-      sparkContext, "number of hash probe input vectors"),
-    "hashProbeInputBytes" -> SQLMetrics.createSizeMetric(
-      sparkContext, "number of hash probe input bytes"),
-    "hashProbeRawInputRows" -> SQLMetrics.createMetric(
-      sparkContext, "number of hash probe raw input rows"),
-    "hashProbeRawInputBytes" -> SQLMetrics.createSizeMetric(
-      sparkContext, "number of hash probe raw input bytes"),
     "hashProbeOutputRows" -> SQLMetrics.createMetric(
       sparkContext, "number of hash probe output rows"),
     "hashProbeOutputVectors" -> SQLMetrics.createMetric(
       sparkContext, "number of hash probe output vectors"),
     "hashProbeOutputBytes" -> SQLMetrics.createSizeMetric(
       sparkContext, "number of hash probe output bytes"),
```

Comment:
looks it's broken. let's remove it.
<html>
<body>
<!--StartFragment-->

number of hash probe output bytes | -39,682,000,181,513,076,736 (495,594,061,042,182,848.0)
-- | --


<!--EndFragment-->
</body>
</html>